deploy/playbooks/examples/deploy_prod.yml

```diff
@@ -36,5 +35,5 @@
      callback_user: ""
      callback_key: ""
 
-     nginx_ssl_cert_path: "/etc/ssl/certs/{{ fqdn }}-bundled.crt"
-     nginx_ssl_key_path: "/etc/ssl/private/{{ fqdn }}.key"
+     nginx_ssl_cert_path: "/etc/letsencrypt/live/{{ fqdn }}/fullchain.pem"
```

Comment:
chacra.ceph.com had its own non-letsencrypt SSL cert when it was set up but at some point, I (or someone) converted it to use a letsencrypt encrypt cert.